CMakeLists.txt

```diff
@@ -43,6 +43,11 @@ IF(HAS_NO_UNDEFINED)
   SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wl,--no-undefined")
 ENDIF()
 
+# OpenMP is required by KNITRO
+FIND_PACKAGE(OpenMP)
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+    SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
+
```

Comment:
Wrong indent, and @thomas-moulard removed that [a while ago](https://github.com/roboptim/roboptim-core-plugin-knitro/commit/524cad94fbcac1bd1e1e4b35b5f3520edf21bb0b). Was it fixed since then?